genesyscloud/resource_genesyscloud_flow_test.go

```diff
@@ -111,17 +112,19 @@ func TestAccResourceFlowForceUnlock(t *testing.T) {
 	})
 }
 
-func TestAccResourceStandardFlow(t *testing.T) {
+func TestAccResourceArchFlowStandard(t *testing.T) {
+	myDir, _ := os.Getwd()
+
 	var (
 		flowResource1 = "test_flow1"
 		flowResource2 = "test_flow2"
 		flowName1     = "Terraform Flow Test-" + uuid.NewString()
 		flowName2     = "Terraform Flow Test-" + uuid.NewString()
 		flowType1     = "INBOUNDCALL"
 		flowType2     = "INBOUNDEMAIL"
-		filePath1     = "../examples/resources/genesyscloud_flow/inboundcall_flow_example.yaml"
-		filePath2     = "../examples/resources/genesyscloud_flow/inboundcall_flow_example2.yaml"
-		filePath3     = "../examples/resources/genesyscloud_flow/inboundcall_flow_example3.yaml"
+		filePath1     = myDir + "/../examples/resources/genesyscloud_flow/inboundcall_flow_example.yaml" //Have to use an explicit path because the filesha function gets screwy on relative class names
```

Comment:
if you got a relative path, go'll resolve it, check out the func where golang resolves it

https://pkg.go.dev/path/filepath#Abs

or you can still do this, but should probably use path.join() instead of concatenation.